bin/cron/send_workshop_reminder_emails

```diff
@@ -1,26 +1,44 @@
 #!/usr/bin/env ruby
+
+# This script sends reminder via poste for professional development workshop
+# signups.
+
 require File.expand_path('../../../pegasus/src/env', __FILE__)
-require 'cdo/hip_chat'
-require 'cdo/solr'
 require 'cdo/poste'
 
-SOLR = Solr::Server.new(host: CDO.solr_server) if CDO.solr_server
-
-def main
-  workshops = SOLR.query(q: "*:*", fq: ["kind_s:ProfessionalDevelopmentWorkshop", "first_date_dt:[NOW/DAY+12DAY TO NOW/DAY+13DAY]"], sort: "id asc", rows: 1000)
-  workshops.each do |workshop|
-    signups = SOLR.query(q: "*:*", fq: ["kind_s:ProfessionalDevelopmentWorkshopSignup", "workshop_id_i:#{workshop['id']}", "-status_s:cancelled"], sort: "id asc", rows: 1000)
-    signups.each do |signup|
-      send_reminder_email(signup)
-      break
-    end
-  end
+# @param date_as_string [String] a date string in the form MM/DD/YY.
+# @returns [Date | nil] a Date for date_as_string or nil if it does not parse
+#   to a date.
+def parse_workshop_date(date_as_string)
+  Date.strptime(date_as_string, '%m/%d/%y')
+rescue ArgumentError
+  nil
```

Comment:
This `rescue ArgumentError` is necessary. Based on logging (without investigation), I'm seeing `date_as_string` (e.g., `parsed_data['dates'].first['date_s']`) can be empty.